.github/workflows/test.yml

```diff
@@ -279,28 +283,45 @@ jobs:
 
     timeout-minutes: 20
     steps:
-      - name: Download build artifacts
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+
+      - name: Add GNU tar to PATH (significantly faster than windows tar)
+        if: matrix.flavor.target == 'windows'
+        run: echo "C:\Program Files\Git\usr\bin" >> $GITHUB_PATH
+
+      - name: Download build artifacts for ${{ matrix.flavor.target }}
         uses: actions/download-artifact@v4
         with:
           name: build-artifacts-${{ matrix.flavor.target }}
-          path: /usr/local/bin
+          path: ${{ github.workspace }}
 
-      - name: Set execute permissions on atmos
-        run: chmod +x /usr/local/bin/atmos
+      - name: Add build artifacts directory to PATH for ${{ matrix.flavor.target }}
+        run: |
+          ls "${{ github.workspace }}"
+          echo "${{ github.workspace }}" >> $GITHUB_PATH
 
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v4
+      - name: Set execute permissions on atmos for ${{ matrix.flavor.target }}
+        if: matrix.flavor.target == 'linux' || matrix.flavor.target == 'macos'
+        run: chmod +x "${{ github.workspace }}/atmos"
 
       - uses: hashicorp/setup-terraform@v3
         with:
           terraform_version: ${{ env.TERRAFORM_VERSION }}
           terraform_wrapper: false
 
-      - name: Run tests for ${{ matrix.demo-folder }}
+      - name: Run tests for ${{ matrix.demo-folder }} with ${{ matrix.flavor.target }}
+        working-directory: examples/${{ matrix.demo-folder }}
+        if: matrix.flavor.target == 'linux' || matrix.flavor.target == 'macos'
         run: |
-          cd examples/${{ matrix.demo-folder }}
           atmos test
 
+      - name: Run tests for ${{ matrix.demo-folder }} with ${{ matrix.flavor.target }}
+        working-directory: examples/${{ matrix.demo-folder }}
+        if: matrix.flavor.target == 'windows'
+        run: |
+          atmos.exe test
```

Comment:
@coderabbitai when this workflow runs, atmos.exe is not found despite path being set. Any ideas?